segway/gmtk/params.py

```diff
@@ -250,13 +333,13 @@ def convert(
     return cls(value)
 
 
-class DPMF(OneLineKind):
+class DPMF(OneLineArray):
     """
     A single DPMF object.
     """
     kind = OBJ_KIND_DPMF
 
-    # todo check if sums to 1.0
+    # XXX: check if sums to 1.0
```

Comment:
do this

Reply:
Used a similar `__new__` method to verify the sum and initialize the `pseudocount` attribute.